flake.nix

```diff
@@ -119,7 +119,17 @@ rec {
             export NIX_STATE_DIR=$TMPDIR
 
             rm -f site-styles/common-styles
-            ln -s ${nixos-common-styles.packages."${system}".commonStyles} site-styles/common-styles
```

Comment:
Since we will have to repeat this for status.nixos.org it might be good to distribute this functions with `nixos-common-styles`